### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by Bodigrim**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/other.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+name: Other questions
+about: Governance of core libraries other than `base`; CLC policies and administration.
+title: ''
+labels: ''
```
Comment (mixphix) [inline .github/ISSUE_TEMPLATE/other.md:5]: Perhaps automatically add "meta" label? Whatever the one we already have is.

**Review by mixphix — APPROVED** [review]

**Comment by tomjaguarpaw** [comment]
Looks fine, though I'm confused why they're named `ISSUE_TEMPLATE` rather than something to do with "pull request".

File: .github/ISSUE_TEMPLATE/other.md (hunk 1 above)
Comment (Bodigrim) [reply .github/ISSUE_TEMPLATE/other.md:5]: There are two possible labels: "meta" and "core libraries". I can split this template into two, if you feel it's worth doing. 

The purpose is not that much about automatic label assignment, but to make sure that people do read `PROPOSALS.md` and understand expectations.

Comment (mixphix) [reply .github/ISSUE_TEMPLATE/other.md:5]: I do think meta- and other-core-library questions should be triaged, but I don't expect there to be an overwhelming volume.

**Comment by tomjaguarpaw** [comment]
I guess `other.md` *is* for issues, but I still don't get why `proposal.md` is not (or seems not to be) for PRs.

**Comment by Bodigrim** [comment]
@tomjaguarpaw on contrast to GHC proposals, CLC proposals are normally issues, not PRs.

**Review by hasufell — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/other.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+assignees: ''
+---
+
+<!--
+Thanks for raising an issue.
+
+Before proceeding, please make sure that you read [`README.md`](https://github.com/haskell/core-libraries-committee#readme) and [`PROPOSALS.md`](https://github.com/haskell/core-libraries-committee/blob/main/PROPOSALS.md).
+
+Do not use this template to discuss changes to `base` package, use [this one](https://github.com/haskell/core-libraries-committee/issues/new?template=proposal.md) instead.
```
Comment (konsumlamm) [inline .github/ISSUE_TEMPLATE/other.md:14]:
```suggestion
Do not use this template to discuss changes to the `base` package, use [this one](https://github.com/haskell/core-libraries-committee/issues/new?template=proposal.md) instead.
```

File: .github/ISSUE_TEMPLATE/proposal.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+name: CLC proposal
+about: Raise a new CLC proposal to change `base` package.
```
Comment (konsumlamm) [inline .github/ISSUE_TEMPLATE/proposal.md:3]:
```suggestion
about: Raise a new CLC proposal to change the `base` package.
```

**Comment by tomjaguarpaw** [comment]
Ah yes, oops, that explains it! Thanks.

**Review by chshersh — APPROVED** [review]
This is definitely an improvement! 👏🏻

File: .github/ISSUE_TEMPLATE/proposal.md (hunk 3 above)
Comment (Bodigrim) [reply .github/ISSUE_TEMPLATE/proposal.md]: Thanks!

File: .github/ISSUE_TEMPLATE/proposal.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+about: Raise a new CLC proposal to change the `base` package.
+title: ''
+labels: ''
+assignees: ''
+---
+
+<!--
+Thanks for raising a new CLC proposal.
+
+Before proceeding, please make sure that you read and understood [`README.md`](https://github.com/haskell/core-libraries-committee#readme) and [`PROPOSALS.md`](https://github.com/haskell/core-libraries-committee/blob/main/PROPOSALS.md).
```
Comment (mixphix) [inline .github/ISSUE_TEMPLATE/proposal.md:12]:
```suggestion
Before proceeding, please make sure that you read and understand [`README.md`](https://github.com/haskell/core-libraries-committee#readme) and [`PROPOSALS.md`](https://github.com/haskell/core-libraries-committee/blob/main/PROPOSALS.md).
```

File: .github/ISSUE_TEMPLATE/other.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+about: Governance of core libraries other than `base`; CLC policies and administration.
+title: ''
+labels: ''
+assignees: ''
+---
+
+<!--
+Thanks for raising an issue.
+
+Before proceeding, please make sure that you read [`README.md`](https://github.com/haskell/core-libraries-committee#readme) and [`PROPOSALS.md`](https://github.com/haskell/core-libraries-committee/blob/main/PROPOSALS.md).
```
Comment (mixphix) [inline .github/ISSUE_TEMPLATE/other.md:12]:
```suggestion
Before proceeding, please make sure that you read and understand [`README.md`](https://github.com/haskell/core-libraries-committee#readme) and [`PROPOSALS.md`](https://github.com/haskell/core-libraries-committee/blob/main/PROPOSALS.md).
```

**Review by mixphix — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/other.md (hunk 5 above)
Comment (Bodigrim) [reply .github/ISSUE_TEMPLATE/other.md:12]: Not quite: it's fair to ask a question if you read but do not understand ;)

**Review by mixphix — APPROVED** [review]

**Comment by Bodigrim** [comment]
I'll go ahead with this, feel free to send suggestions if any later.

**Pull request merged**